persistence/src/commonMain/db_user/com/wire/kalium/persistence/ConversationDetailsWithEvents.sq

```diff
@@ -65,6 +66,18 @@ LEFT JOIN MessageConversationChangedContent AS ConversationNameChangedContent
     ON LastMessage.message_id = ConversationNameChangedContent.message_id AND LastMessage.conversation_id = ConversationNameChangedContent.conversation_id
 LEFT JOIN MessageAssetContent AS AssetContent
     ON LastMessage.message_id = AssetContent.message_id AND LastMessage.conversation_id = AssetContent.conversation_id
+LEFT JOIN (
+    SELECT
+        message_id,
+        conversation_id,
+        CASE
+            WHEN COUNT(DISTINCT asset_mime_type) = 1 THEN MIN(asset_mime_type)
+            ELSE NULL
+        END AS asset_mime_type,
+        COUNT(*) attachments_count
+    FROM MessageAttachments
+    GROUP BY message_id, conversation_id
+) AS AttachmentsContent ON LastMessage.message_id = AttachmentsContent.message_id AND LastMessage.conversation_id = AttachmentsContent.conversation_id
```

Comment:
can you please provide some benchmark info about the effect of this join when there are multiple conversations with many assets as last message